app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -96,9 +97,9 @@ class SettingsFragment : PreferenceFragmentCompat(), AccountObserver {
         preferenceManager.sharedPreferences.registerOnSharedPreferenceChangeListener(preferenceChangeListener)
 
         if (SDK_INT <= Build.VERSION_CODES.M) {
-            findPreference<DefaultBrowserPreference>(getPreferenceKey(pref_key_make_default_browser))?.apply {
```

Comment:
I think all we need to do is replace `DefaultBrowserPreference` with `Preference`? It looks like we just replaced this in `preferences.xml` and forgot to update it here

Reply:
AFAIK `Preference` does not have an `isVisible` property or a method to hide it. That is why I used `removePreference` from it's parent category. I will check on Monday though if you prefer that way.

Reply:
I'm pretty sure that will work. I may just push a quick patch to this PR so we don't have crashes over the weekend, if you don't mind!

Reply:
It's fine by me! :)